projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/Untitled.ipynb

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
In this accidentally pushed ?

Reply:
seems that way, I deleted a bunch of extra empty notebooks but I must have missed this one